packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -2897,30 +2885,114 @@ class StackViewport extends Viewport implements IStackViewport, IImagesLoader {
   public isReferenceViewable(
     viewRef: ViewReference,
     options: ReferenceCompatibleOptions = {}
-  ): boolean {
+  ): boolean | unknown {
```

Comment:
This seems really ugly to change the signature on an 'is' method.  I'd rather return !!value to convert to a boolean.